disputer/truffle-config.js

```diff
@@ -0,0 +1,4 @@
+const path = require("path");
+const wkdir = path.dirname(require.resolve("@umaprotocol/core/package.json"));
```

Comment:
What does `require.resolve` do?

Reply:
I really like the way they describe it in the nodejs docs: "Uses the internal require() machinery to look up the location of a module, but rather than loading the module, just returns the resolved filename."

Reply:
To put it differently, it basically tells you the exact path to a module rather than reading the module.

Side note: the reason `package.json` is used is that it's a file that should always exist within all modules and isn't dependent on a specific entrypoint or index file existing.